charts/elastic/1.2.1/values.yaml

```diff
@@ -0,0 +1,264 @@
+        affinity:
+          podAntiAffinity:
+            preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchLabels:
+                    elasticsearch.type: data
+                topologyKey: kubernetes.io/hostname
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: kubernetes.io/os
+                  operator: In
+                  values:
+                  - linux
+#                - key: elastic
+#                  operator: In
+#                  values:
+        tolerations: []
+      ml:
+        autoscaling:
+          enabled: false
+          max_nodes: 2
+          max_cpu: 2000m
+          max_memory: 4Gi
+          max_storage: 20Gi
+        count: 1
+        cpu: 2000m
+        memory: 4Gi
+        storage: 20Gi
+        roles: '["ml"]'
+        labels:
+          elasticsearch.type: data
+        affinity:
+          podAntiAffinity:
+            preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchLabels:
+                    elasticsearch.type: data
+                topologyKey: kubernetes.io/hostname
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: kubernetes.io/os
+                  operator: In
+                  values:
+                  - linux
+#                - key: elastic
+#                  operator: In
+#                  values:
+        tolerations: []
+    storageClass: ""
+    image:
+      registry: docker.elastic.co
+      repository: elasticsearch/elasticsearch
+      tag: ""
+    
+    ingress:
+      enabled: true
+      ingressClassName: ""
+      hostname: es-tst.apps.example.com
+      annotations:
+        nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+        nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+        nginx.ingress.kubernetes.io/ssl-redirect: "true"
+        nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+        nginx.ingress.kubernetes.io/proxy-body-size: "0"
+        nginx.ingress.kubernetes.io/proxy-read-timeout: "600"
+        nginx.ingress.kubernetes.io/proxy-send-timeout: "600"
+    service:
+      annotations: {}
+
+packageRegistry:
+  enabled: true
+  params:
+    image:
+      registry: docker.elastic.co
+      repository: package-registry/distribution
+      tag: ""
+    ingress:
+      enabled: true
+      ingressClassName: ""
+      hostname: repo-tst.apps.example.com
+      annotations: {}
+    service:
+      annotations: {}
+
+kibana:
+  enabled: true
+  params:
+    autoscaling:
+      enabled: false
+      maxReplicas: 2
+      CPUaverageUtilization: 50
```

Comment:
```suggestion
      maxNodes: 2
      averageCpuUtilization: 50
```

Reply:
Using key names for kibana from elastic documentation:
https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-stateless-autoscaling.html

Reply:
Corrected to:
averageUtilization